.github/issue_template.md

```diff
@@ -0,0 +1,40 @@
+Note: This is just a template, so feel free to use/remove the unnecessary things
+
+### Description
+- Type: Bug | Enhancement\Feature Request | Question
+- Priority: Blocker | Major | Minor
+
+---------------------------------------------------------------
+## Bug
+
+**OS**  
+linux|windows|??
+
+**mbed TLS build:**  
+Version: x.x.x or git commit id  
+Configuration: please attach config.h file  
```

Comment:
I don't think we should make `config.h` mandatory. Some people may not want to share it, or may not know it. We should make it optional, so I suggest saying 'please attach config.h file where possible'.